File: async-scala-client/src/main/scala/io/swagger/client/model/StateTransition.scala
package io.swagger.client.model

import org.joda.time.DateTime
import java.util.UUID


case class StateTransition (
  _type: String,  // - &#39;voided&#39; - &#39;unvoided&#39; - &#39;reconciled&#39; - &#39;unreconciled&#39; - &#39;filed&#39; - &#39;unfiled&#39; - &#39;filedByAvalara&#39; 
comment: Option[String]
)

File: javascript-closure-angular-client/API/Client/StateTransition.js
goog.provide('API.Client.StateTransition');

/**
 * @record
 */
API.Client.StateTransition = function() {}

/**
 * - 'voided' - 'unvoided' - 'reconciled' - 'unreconciled' - 'filed' - 'unfiled' - 'filedByAvalara' 
 * @type {!string}
 * @export
 */
API.Client.StateTransition.prototype.type;

/**
 * @type {!string}
 * @export
 */
API.Client.StateTransition.prototype.comment;

/** @enum {string} */
API.Client.StateTransition.TypeEnum = { 
  voided: 'voided',
  unvoided: 'unvoided',
  reconciled: 'reconciled',
  unreconciled: 'unreconciled',
  filed: 'filed',
  unfiled: 'unfiled',
  filedByAvalara: 'filedByAvalara',
}
